pkg/schema/schema.go

```diff
@@ -2,36 +2,38 @@
 
```

Comment:
can you please also update the README with the new possibility to "pre-set" those values?